[{ "dataset" : ["cifar10"], "net" : ["lenet_cifar"], "n_clients" : [1000], "n_data" : [null], "communication_rounds" : [1000], "local_epochs" : [1, 3, 10], "participation_rate" : [0.01, 0.1], "classes_per_client" : [2], "batch_size" : [128], "balancedness" : [1.0], "pretrained" : [null], "save_model" : [null], "log_frequency" : [-100], "log_path" : ["manyclients_big/"], "job_id" : [$SLURM_JOB_ID] }]
Traceback (most recent call last):
  File "/opt/code/federated_learning.py", line 116, in <module>
    run()
  File "/opt/code/federated_learning.py", line 101, in run
    experiments_raw = json.loads(args.hyperparameters)
  File "/opt/conda/lib/python3.6/json/__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "/opt/conda/lib/python3.6/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/opt/conda/lib/python3.6/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 21 column 14 (char 409)
